contrib/devtools/github-merge.py

```diff
@@ -209,7 +209,7 @@ def main():
             exit(4)
 
         symlink_files = get_symlink_files()
-        for f in symlink_files;
+        for f in symlink_files:
```

Comment:
This commit (9c8c12e) title should probably say smth like `Merge bitcoin#XXXX: devtools: Fix a syntax error typo`

Reply:
original commit was not a merge, but a commit: `309bf1625 devtools: Fix a syntax error typo`